svc/cli/cmd/remote_check.go

```diff
@@ -0,0 +1,76 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	native "net/http"
+
+	"github.com/getapid/apid-cli/common/config"
+	"github.com/getapid/apid-cli/common/result"
+	"github.com/getapid/apid-cli/common/step"
+	"github.com/getapid/apid-cli/common/transaction"
+	"github.com/getapid/apid-cli/common/variables"
+	cmdResult "github.com/getapid/apid-cli/svc/cli/result"
+	"github.com/spf13/cobra"
+)
+
+var (
+	apiKey = ""
+	region = ""
+)
+
+var remoteCmd = &cobra.Command{
+	Use:   "remote",
+	Short: "Executes a config on apid remote infrastructure",
+	Long: `Loads either a config file, or if a directory is provided it 
+recursively loads all .yaml files and executes them. It will run all the 
+transactions in you config, verify the responses, and record the time it 
+took to action each request. Runs all steps in each transaction on a 
+public cloud infrastructure.`,
+	Example: `
+	apid remote	--key <apid access key>
+	apid remote --config my-api.yaml --key <apid access key> --region us-east
+	apid remote -c ./e2e-tests/ -k <apid access key>
+	apid remote -c ./e2e-tests/ -k <apid access key> -r us-east`,
+	Args: cobra.NoArgs,
+	RunE: remoteRun,
+}
+
+func init() {
+	rootCmd.AddCommand(remoteCmd)
+	remoteCmd.Flags().StringVarP(&configFilepath, "config", "c", "./apid.yaml", "file with config to run")
+	remoteCmd.Flags().BoolVarP(&showTimings, "timings", "t", false, "output the durations of requests")
+	remoteCmd.Flags().StringVarP(&apiKey, "key", "k", "", "apid access key")
```

Comment:
we should (also) get this from the environment (amend docs when changed)

Reply:
Changed code and docs.